funding/manager.go

```diff
@@ -261,6 +261,17 @@ type InitFundingMsg struct {
 	// peer.
 	MaxLocalCsv uint16
 
+	// FundUpToMaxAmt is the maximum amount to try to commit to. If set, the
+	// FundMinAmt field denotes the acceptable minimum amount to
+	// commit to, while trying to commit as many coins as possible up to
+	// this value.
+	FundUpToMaxAmt btcutil.Amount
+
+	// FundMinAmt denotes the minium amount to commit to. If set, the
+	// FundUpToMaxAmt field denotes the maximum amount to commit to, while
+	// trying to commit as many coins as possible up to this value.
```

Comment:
nit: I think this comment is a bit off, by "this value" I think the FundUpToMaxAmt is referred to, but its the description for the FundMinAmt.

Reply:
Yep, good catch